features/cursor-style.yml

```diff
@@ -0,0 +1,47 @@
+name: Cursor Style
+description: The `cursor` CSS property styles the pointer, allowing you to provide hints to the user on how to interact with the hovered element.
+spec: https://drafts.csswg.org/css-ui-4/#cursor
+caniuse:
+  - css3-cursors
+  - css3-cursors-newer
```

Comment:
Depending on the results of https://github.com/mdn/browser-compat-data/issues/24142, we may want to include https://caniuse.com/css3-cursors-grab too.